interface-definitions/firewall.xml.in

```diff
@@ -314,6 +314,85 @@
           </tagNode>
         </children>
       </node>
+      <tagNode name="interfaces">
+        <properties>
+          <help>Interface name</help>
+          <completionHelp>
+            <script>${vyos_completion_dir}/list_interfaces.py</script>
+          </completionHelp>
+        </properties>
+        <children>
+          <node name="in">
+            <properties>
+              <help>Forwarded packets on inbound interface</help>
+            </properties>
+            <children>
+              <leafNode name="name">
```

Comment:
Please create an xml.i file so we can simply reference the `name` and `ipv6-name` CLI node.